lib/bitbucket_rest_api/repos.rb

```diff
@@ -63,7 +66,15 @@ def sources
     def services
       @services ||= ApiFactory.new 'Repos::Services'
     end
-
+    def forks
+      @services ||= ApiFactory.new 'Repos::Forks'
+    end
+    def commits
+      @services ||=ApiFactory.new 'Repos::Commits'
```

Comment:
Surrounding space missing for operator '||='.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.